docs/reference/query-languages/esql/_snippets/functions/layout/match.md

```diff
@@ -2,12 +2,13 @@
 
 ## `MATCH` [esql-match]
 :::{warning}
-###### COMING 9.1.0
-
 Do not use on production environments. This functionality is in technical preview and
 may be changed or removed in a future release. Elastic will work to fix any issues, but features in technical preview
 are not subject to the support SLA of official GA features.
+:::
 
+:::{note}
+###### COMING 9.1.0
```

Comment:
I just want to say that this looks less confusing to me than having the `COMING 9.1.0` inside the tech preview warning.

Reply:
💯